rotkehlchen/db/reports.py

```diff
@@ -226,14 +227,21 @@ def add_report_data(
         - InputError if the event can not be written to the DB. Probably report id does not exist.
         """
         data = event.serialize_for_db(ts_converter)
+
+        pnl_taxable = str(event.pnl.taxable)
+        try:
+            asset = event.asset.symbol_or_name()
```

Comment:
when sorting do we send the symbol? make sure that the schema sorts by symbol and not id

Reply:
Actually is only the [timestamp column](https://github.com/rotki/rotki/blob/develop/frontend/app/src/components/profitloss/ProfitLossEvents.vue#L50) used in frontend, I don't see a [column of asset in the table headers ](https://github.com/rotki/rotki/blob/develop/frontend/app/src/components/profitloss/ProfitLossEvents.vue#L57)but the [api return the asset id](https://github.com/rotki/rotki/blob/develop/rotkehlchen/accounting/structures/processed_event.py#L127) and [the symbol](https://github.com/rotki/rotki/blob/develop/docs/api.rst#get-saved-events-of-a-pnl-report) for my not have problem in leaving the symbol because we can sort is more accurate to the use for example have the token ABC with the id 2 and the token ACB with the id so sorting will appear that it's incorrect

Reply:
Actually is only the[ timestamp column used](https://github.com/rotki/rotki/blob/develop/frontend/app/src/components/profitloss/ProfitLossEvents.vue#L50) in frontend for sort, and I think the column of asset not is used . Also the [api only returns the asset id](https://github.com/rotki/rotki/blob/develop/rotkehlchen/accounting/structures/processed_event.py#L127) but it will sort by the column and save the id on the asset column can cause the issue eg asset with symbol "acb" with ID 1 and another with symbol "abc" with ID 2